### PR TITLE
errorMSG

### PR DESCRIPTION
**Pull request by J-VUK**
Entering "VapeBot Delivery Information" causes an error.

### CODE REVIEW

File: errorMSG [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Hey, sorry for the awkward way of letting you know, but I found an error on minimalist.
```
Comment (mshwery) [inline errorMSG:1]: No worries! :) 

This isn't the right repo (it isn't public) but I'll improve the error messages so they are easier for people to understand.

**Comment by mshwery** [comment]
Closing as unrelated to this repo.

**Comment by J-VUK** [comment]
You are welcome

01273 964 252
***@***.***
vapeuk.co.uk
Unit 2A, Victoria Trading Estate, Victoria Road, Portslade, East Sussex, BN41 1XQ
Vape UK is a trading name of C Page Ltd. Registered Office: Unit 2A, Victoria Trading Estate, Victoria Road, Portslade, East Sussex, BN41 1XQ. Registered in England No: 5381836 VAT No: 855 5588 77.
​
​Please consider the environment before printing this email. Computer viruses can be transmitted via email. The recipient should check this email and any attachments for the presence of viruses. Vape UK accepts no liability for any damage caused by any virus transmitted by this email. Vape UK accepts no liability for the content of this email, or for the consequences of any actions taken on the basis of the information provided, unless that information is subsequently confirmed in writing. This email and any files transmitted with it are confidential and intended solely for the use of the individual or entity to whom they are addressed. If you have received this email in error, please notify the system manager. This message contains confidential information and is intended only for the individual named. If you are not the named addressee, you should not disseminate, distribute or copy this email. Vape UK is 100% compliant with the GDPR data protection regulation. Visit our privacy policy to learn how we collect, keep, and process your private information in accordance with these laws. Vape UK Privacy Policy

On 19 Sep 2021, at 18:50, Matt Shwery ***@***.***> wrote:

﻿

Closed #102<https://protect-eu.mimecast.com/s/RBbACPYv6TP5Vs0DH8f?domain=github.com>.

**Comment by mshwery** [comment]
One thing to note: this is expected because there is a limit on the name length. At the time, it was 21 characters for a list name, but I've increased it to a max of 60 characters, which would allow for `VapeBot Delivery Information`

**Comment by darkks** [comment]
But is the code within the error message necessary?

**Pull request closed without merging**